docs/src/appendix/sncast/verify.md

```diff
@@ -4,18 +4,25 @@ Verify Cairo contract on a chosen verification provider.
 ## `--contract-address, -a <CONTRACT_ADDRESS>`
 Required.
 
-The address of the contract that is to be verified.
+The address of the contract that is to be verified. Either this or the class hash should be provided.
 
-## `--contract-name <CONTRACT_NAME>`
+
+## `--class-hash, -c <CLASS_HASH>`
+Required.
+
+The class hash of the contract that is to be verified. Either this or the contract address should be provided.
+
+## `--class-name <CLASS NAME>`
 Required.
 
-The name of the contract. The contract name is the part after the `mod` keyword in your contract file.
+The name of the contract class. The contract name is the part after the `mod` keyword in your contract file.
```

Comment:
Now I think that maybe it should say under `Required.`: `Conflicts with: ...`. instead of `Either ...`. It's more explicit I think.